src/Microsoft.AspNetCore.Authentication/RemoteAuthenticationOptions.cs

```diff
@@ -56,6 +56,11 @@ public string DisplayName
         /// </summary>
         public bool SaveTokensAsClaims { get; set; }
 
+        /// <summary>
+        /// Gets or sets the lifetime of the correlation cookie used to prevent cross-site scripting attacks.
```

Comment:
That's why I suggested the ridiculous name that I did. Any ideas for a property name that conveys this? There's no reason for this timeout usage to be restricted to the correlation cookie. Some derivations of RemoteAuthenticationHandler might use mechanics other than correlation cookies. E.g. how about applying this to Twitter's State cookie?